### PR TITLE
0.12rc compiler fixes

### PR DESCRIPTION
**Pull request by justinwoo**
Could we have a compiler/0.12 branch made and move this in there for now?

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -14,14 +14,14 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-maybe": "^3.0.0",
-    "purescript-functions": "^3.0.0"
+    "purescript-maybe": "purescript/purescript-maybe#compiler/0.12",
```
[the remark this answers is not in the document]
Comment (justinwoo) [reply bower.json]: hasn't worked for me

[the remark this answers is not in the document]
Comment (justinwoo) [reply bower.json]: ha, bower requires `#` in front of branch names since it won't try to figure it out

**Comment by justinwoo** [comment]
ping, this is updated

**Comment by kRITZCREEK** [comment]
compiler/0.12 already has all these changes. I guess Gary decided it was easier to do them himself %%

**Pull request closed without merging**